src/main/java/uk/gov/justice/hmpps/prison/api/model/TransactionHistory.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import lombok.Builder;
+
+import java.util.Collections;
+import java.util.List;
+
+@Builder
+public class TransactionHistory {
```

Comment:
Not sure what this gives us. You could just have a TransactionHsitoryReposiutory that returns TransactionHistoryItem or  TransactionHistory.